src/assets/translations/fr/main.json

```diff
@@ -1757,11 +1759,16 @@
       },
       "import": {
         "header": "Importer votre porte-monnaie",
+        "keystoreHeader": "Importer votre porte-monnaie Keystore",
         "body": "Entrez ou collez votre phrase de récupération secrète en minuscules, sans virgule, ni chiffre et avec un seul espace entre chaque mot.",
         "button": "Suivant",
         "secretRecoveryPhraseError": "Saisissez votre phrase secrète de récupération en laissant un seul espace entre les mots. Omettez les virgules, les retours à la ligne ou tout autre caractère supplémentaire.",
         "secretRecoveryPhraseRequired": "La phrase secrète de récupération est requise",
-        "secretRecoveryPhraseTooShort": "La phrase secrète de récupération est trop courte"
+        "secretRecoveryPhraseTooShort": "La phrase secrète de récupération est trop courte",
+        "dragAndDrop": "Glisser-déposer ou Choisir un fichier",
+        "keystoreImportBody": "Téléchargez le fichier en cliquant sur Parcourir ou en le faisant glisser ci-dessous.",
```

Comment:
Prefer "Importez". This is all local, nothing being upload anywhere so could make users scared for no reason.

Reply:
Fair point, but we might need to change this in English first as it's "Upload" 😅 

https://github.com/shapeshift-globalization/web/blob/456f2c3077327d802cda0fd5e33835ac5e9a1d66/src/assets/translations/en/main.json#L1768-L1769

It could be "Choose a file" too instead of Import... 

Translators have already submitted their translators for this, so I think we should do that in the next sync. Sounds good? I can make the PR to update it in English later.

Reply:
Next sync sounds good, and nice catch on the english flavour! Should definitely be "Import" there too.